Base/usr/share/man/man7/Mitigations.md

```diff
@@ -95,10 +95,14 @@ Kernel: Add a basic implementation of unveil()
 `jails` are mitigation originating from FreeBSD.
 It allows a program to be placed inside a lightweight OS-level virtualization environment.
 
-Currently the only isolation is Process ID restricted view of only other processes in the same jail.
-The `/proc` filesystem only exports PID subdirectories of other processes in the same jail, and the same
-happens for exported information in `/sys/kernel/processes`.
-Jailed processes are forbidden write access to kernel variables (which are located in `/sys/kernel/variables`).
+Current restrictions on jailed processes:
+- Process ID view isolation, being limited (both in `/proc` and `/sys/kernel/processes`) to only processes that share the same jail.
+
+Special restrictions on filesystem also apply:
+- Write access is forbidden to the `/sys/kernel/power_state` node.
+- Read accesses is forbidden by default to all nodes in `/sys/kernel` directory, except for:
+    `df`, `interrupts`, `keymap`, `memstat`, `processes`, `stats` and `uptime`.
+- Write access is forbidden to kernel variables (which are located in `/sys/kernel/variables`).
```

Comment:
Could we fixup the last commit here with the first commit? Since the documentation is initially added in this PR, it makes more sense to keep them combined. If it makes more sense we could move the documentation commit to the end of the PR so that it describes the final state of the jails subsystem after the changes in here.